lib/services/product-manager.js

```diff
@@ -112,17 +112,20 @@ export default class ProductManager {
     return this._fetchProductsByPredicate(predicate)
   }
 
-  async _getProductsBySkuOrSlugChunk (skus, slugs = []) {
+  async _getProductsBySkuOrSlugChunk (skus, slugs = [], whiteListedLocales = []) {
```

Comment:
Why do you need a default again if you already do it here: https://github.com/commercetools/commercetools-node-variant-reassignment/pull/88/files#diff-9258e95959eab5fe80e115dd55349fa6a1b85b37392e9fb754bc05da5c5452eeR14

Reply:
I think now it's better.
https://github.com/commercetools/commercetools-node-variant-reassignment/pull/88/commits/c26bb3f5143d0fa896c1f3a2574f9d4343ea7509

Reply:
I think if you want to define a default value then it should be as soon as possible as in the future someone might use the variable sooner and could hit the NPE there.